src/schemas/EnvironmentStatusesWithStages.yaml

```diff
@@ -0,0 +1,6 @@
+type: object
+properties:
+  environment:
+    $ref: '../schemas/Status.yaml'
+  stages:
+    $ref: '../schemas/EnvironmentStatusesStages.yaml'
```

Comment:
You need to create an intermediate file that represents a list of `DeploymentStageWithServiceStatuses`